{{cookiecutter.project_slug}}/.github/workflows/check-image.yml

```diff
@@ -0,0 +1,24 @@
+name: Build and check image
+
+on: [push, pull_request]
+
+jobs:
+  verify-image-build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo content
+        uses: actions/checkout@v2
+      - name: ooil version
+        uses: docker://itisfoundation/ci-service-integration-library:v1.0.4
```

Comment:
please make sure to use the latest version of this image. we are at 2.x.x currently.

Reply:
please search and replace this repo wide